pom.xml

```diff
@@ -39,11 +39,11 @@
         <module>../xoom-lattice-exchange-camel</module>
         <module>../xoom-lattice-exchange-rabbitmq</module>
         <module>../xoom-turbo</module>
-        <module>../xoom-designer</module>
         <module>../xoom-schemata</module>
+        <module>../xoom-designer</module>
         <module>../xoom-build-plugins</module>
         <module>../xoom-telemetry</module>
-        <module>../xoom-examples</module>
+        <!-- <module>../xoom-examples</module> -->
```

Comment:
Why did you have to coment this out?

Reply:
@jakzal I didn't notice that during review. Can you help here @hamzajg?

Reply:
yeah, it was for test purpose, to have fast feedback when running mvn install -Pmodules -Pnative-image..